webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js

```diff
@@ -217,3 +217,29 @@ describe('Without tracer installed', () => {
     expect(enableTracesModalLink).toHaveClass('pf-m-in-progress');
   });
 });
+
+test('verify banner is present for old host page', async (done) => {
+  // Setup autocomplete with mockForemanAutoComplete since we aren't adding /katello
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+
+  const noResults = {
+    total: 0,
+    subtotal: 0,
+    page: 1,
+    per_page: 20,
+    results: [],
+  };
+
+  const scope = nockInstance
+    .get(hostTraces)
+    .reply(200, noResults);
+
+  const { queryByText } = renderWithRedux(<TracesTab />, renderOptions);
```

Comment:
I added an argument to `renderOptions` so that you can tell it if Tracer is installed and it will give you a different Redux initial state based on that:

```suggestion
  const { queryByText } = renderWithRedux(<TracesTab />, renderOptions(true));
```

Reply:
(it may help to rebase on current master since my other Traces PR was just merged.)

Reply:
Updated